front/src/modules/trainschedule/components/Timetable/TimetableToolbar.tsx

```diff
@@ -145,14 +164,37 @@ const TimetableToolbar = ({
         } else {
           dispatch(setFailure(castErrorToFailure(e)));
         }
-      });
+      }
+      // TODO Paced trains : remove the else in https://github.com/OpenRailAssociation/osrd/issues/10791
+    } else {
+      await deleteTrainSchedules({ body: { ids: editoastSelectedTrainScheduleIds } })
+        .unwrap()
+        .then(() => {
+          dispatch(
+            setSuccess({
+              title: t('timetable.trainsSelectionDeletedCount', { count: itemsCount }),
+              text: '',
+            })
+          );
+        })
+        .catch((e) => {
+          if (
+            selectedTrainId &&
+            selectedTimetableItemIds.includes(selectedTrainId as TrainScheduleId)
+          ) {
+            dispatch(updateSelectedTrainId(selectedTrainId));
+          } else {
+            dispatch(setFailure(castErrorToFailure(e)));
+          }
+        });
+    }
   };
 
+  // TODO Paced train : Adapt this to handle export paced trains in issue https://github.com/OpenRailAssociation/osrd/issues/10614
   const exportTrainSchedules = (selectedTrainIdsFromClick: TimetableItemId[]) => {
-    if (!trainSchedules) return;
+    if (!timetableItems) return;
 
-    // TODO Paced train : Adapt this to handle export paced trains in issue https://github.com/OpenRailAssociation/osrd/issues/10614
-    const formattedTrainSchedules = trainSchedules
+    const formattedTrainSchedules = timetableItems
```

Comment:
Should you add a small filter here to keep only the trainSchedules and not the paced trains ?